flake.nix

```diff
@@ -20,7 +20,7 @@
         };
 
         toolchain = with fenix.packages.${system}; fromToolchainFile {
-          file = ./rust-toolchain.toml; # alternatively, dir = ./.;
+          file = ./rust-toolchain; # alternatively, dir = ./.;
           sha256 = "sha256-4vetmUhTUsew5FODnjlnQYInzyLNyDwocGa4IvMk3DM=";
```

Comment:
Perhaps in another PR the sha256 will need to be updated to match Rust 1.70, and the flake may need some other tweaks to add e.g. rust-analyzer back in. cc @johnchandlerburnham

Reply:
I doubt the flake works as is is, see #343 . I didn't dare touch it.